tutorials/Additive_and_multiplicative_bias.ipynb

```diff
@@ -13,12 +13,14 @@
    "source": [
     "Bias measures how well the mean forecast and mean observation correspond to each other. It can tell us whether there is an over or under-forecast tendency and informs how a forecast system could be easily recalibrated.\n",
     "\n",
-    "`Scores` has multiplicative bias and additive bias implementations for use on continuous forecasts.\n",
+    "`Scores` has multiplicative bias, additive bias and additive bias percentage implementations for use on continuous forecasts.\n",
     "\n",
     "Additive bias is often called the \"mean error\". Since it does not tell us the average *magnitude* (i.e. average of the absolute value of the error) of the error, it is possible for there to be little or no bias even when there are large positive and negative errors.\n",
     "\n",
     "Multiplicative bias is well suited for forecasts and observations that have 0 as an upper or lower bound (e.g., significant wave height, or wind magnitude) - or to causes of error which are multiplicative in nature. This could be useful for a forecaster who wants a simple method to bias correct a wind speed forecast as wind speeds can never be negative.\n",
     "\n",
+    "Additive bias percentage is used for evaluating forecast accuracy across stations or dataset with varying magnitudes. By expressing the error as a percentage of the observed value, it allows for standardized comparisons, enabling assessment of forecast performance regardless of the absolute scale of values. Consider three stations with average flow values of 5, 25, and 50 m³/s. Assuming an additive bias of 1 m³/s for each station, the calculated Additive bias percentage values are 20%, 4%, and 2%, respectively. Assuming a threshold of 5% for acceptable performance, the first station (flow value of 5) is considered less accurate, while the other stations demonstrate acceptable performance.\n",
```

Comment:
Should now say "Percent bias is used..."

Where flow values are mentioned, I presume in a hydrology context this will always be the flow of water. In other areas, wind also flows, and perhaps so could other variables. Should the docs be specific and say "water flow" (or another more appropriate term), or explain that PBIAS is generally used in a hydrology context?